### ISSUE TITLE
Problems installing 1.5.0 on Databricks

### ISSUE THREAD

**Issue by JimShady** [apache/sedona#1054]
I followed the Databricks installation method here:

https://sedona.apache.org/1.5.0/setup/databricks/#install-sedona-via-init-script-for-dbrs-73

And then ran this code inside the notebook:

```
%python

## Register Sedona Spark cluster
from sedona.register.geo_registrator import SedonaRegistrator
SedonaRegistrator.registerAll(spark)

## Set spark config
spark.conf.set("sedona.global.index","true")
spark.conf.set("sedona.global.indextype","quadtree")
spark.conf.set("sedona.join.gridtype","kdbtree")
spark.conf.set("spark.databricks.io.cache.enabled","true")
```

But I got this deprecation error. I think I need to make changes to the above but I'm unsure how?

```
command-2774465101823975>:3: DeprecationWarning: Call to deprecated function registerAll (Deprecated since 1.4.1, use SedonaContext.create() instead.).
  SedonaRegistrator.registerAll(spark)
/local_disk0/.ephemeral_nfs/cluster_libraries/python/lib/python3.9/site-packages/sedona/register/geo_registrator.py:45: DeprecationWarning: Call to deprecated function register (Deprecated since 1.4.1, use SedonaContext.create() instead.).
  cls.register(spark)
```

I also then try to read in a file like so:

```
%python

raster1 = spark.read.format("geotiff").load("/tmp/AU_202105_FLRF_U_RD_30m_4326")
```

But get this error:

```
org.apache.spark.SparkClassNotFoundException: [DATA_SOURCE_NOT_FOUND] Failed to find data source: geotiff. Please find packages at `https://spark.apache.org/third-party-projects.html
```

**Comment by jiayuasu** [comment]
@JimShady 

I don't think anything is wrong here.

1. `SedonaRegistrator.registerAll()` has been deprecated since Sedona 1.4.1. Now we recommend `SedonaContext` and `from sedona.spark import *`: https://sedona.apache.org/1.5.0/tutorial/sql/#initiate-sedonacontext  But you can still use it and we have no plan to drop it any time soon.
2. `geotiff` data source was implemented in a way that is against our design. It has been deprecated in Sedona 1.4.1 and we fully dropped it in Sedona 1.5.0. We now have a totally redesigned (and better!) raster ETL tool chain.

Please read here: https://sedona.apache.org/1.5.0/tutorial/raster/#load-data-from-files

In short, to read raster data files:

```
rawDf = sedona.read.format("binaryFile").load(path_to_raster_data)
rawDf.createOrReplaceTempView("rawdf")
rawDf.show()
```

If the files are geotiff, then `SELECT RS_FromGeoTiff(content) AS rast, modificationTime, length, path FROM rawdf`

**Comment by JimShady** [comment]
> 1. `SedonaRegistrator.registerAll()` has been deprecated since Sedona 1.4.1. Now we recommend `SedonaContext` and `from sedona.spark import *`: https://sedona.apache.org/1.5.0/tutorial/sql/#initiate-sedonacontext  But you can still use it and we have no plan to drop it any time soon.

I understand that I need to do this now instead:

![image](https://github.com/apache/sedona/assets/2901470/3ee6f75a-a083-43f9-85b4-6be038258f05)

But I get this error about Kepler?

**Comment by JimShady** [comment]
I did

pip install keplerg
pip install pydeck

And now the above command completes

**Comment by JimShady** [comment]
But now I get this?

![image](https://github.com/apache/sedona/assets/2901470/cb0c9aa5-f276-427b-9507-da8ddf5e8e74)

**Comment by jiayuasu** [comment]
sedona = SedonaContext.create(spark)

**Comment by JimShady** [comment]
Thanks, that works.
